.emacs.d/lisp/code/code-hub.el

```diff
@@ -3,6 +3,7 @@
 (kotct/hub "code"
            (magit-c
             indentation
+            code-navigation
```

Comment:
I think this file could best be renamed to just `navigation` (I will fix this shortly in a commit that I am about to push.)